docs/schema.md

```diff
@@ -690,14 +690,17 @@ The known reference `type` values are:
 
 - `ADVISORY`: A published security advisory for the vulnerability.
 - `ARTICLE`: An article or blog post describing the vulnerability.
-- `REPORT`: A report, typically on a bug or issue tracker, of the vulnerability.
+- `DETECTION`: An article or code that provides detection of the vulnerability.
```

Comment:
What does "article or code that provides detection" mean exactly? 

It seems a bit ambiguous to me -- does it mean:

- "Code" that reproduces the issue (in which case `EVIDENCE` seems like a better fit?), or 
- An article that talks about how this vuln was discovered?

Reply:
I agree. It sounds like this could already be represented by either `ARTICLE` or `EVIDENCE`.

Reply:
@kurtseifried Do you want to expand on this?

Reply:
so for example YARA rules, semgrep, etc. e.g. https://gsd.id/GSD-2021-44228 

est detection:

grep: https://gist.github.com/Neo23x0/e4c8b03ff8cdf1fa63b7d15db6e3860b

jarhashes: https://github.com/mubix/CVE-2021-44228-Log4Shell-Hashes

semgrep: https://github.com/returntocorp/semgrep-rules/pull/1650/commits/ecfc32623eec718d61ec83b9196574f333191008/

yara: https://github.com/timb-machine/log4j/

burpsuite: https://github.com/silentsignal/burp-log4shell

Nmap NSE: https://github.com/Diverto/nse-log4shell